packages/arb-bridge-peripherals/contracts/tokenbridge/ethereum/gateway/L1ArbitrumGateway.sol

```diff
@@ -285,6 +288,28 @@ abstract contract L1ArbitrumGateway is L1ArbitrumMessenger, TokenGateway, ERC165
         return abi.encode(seqNum);
     }
 
+    function outboundTransferCustomRefundWithPermit(
```

Comment:
lets add some natspec explaining the context around this function
https://docs.soliditylang.org/en/develop/natspec-format.html